src/server/game/Spells/SpellInfo.cpp

```diff
@@ -372,6 +372,7 @@ SpellImplicitTargetInfo::StaticData  SpellImplicitTargetInfo::_data[TOTAL_SPELL_
 };
 
 SpellEffectInfo::SpellEffectInfo(SpellInfo const* spellInfo, SpellEffectEntry const* _effect)
+    : EffectAttributes(EFFECT_ATTRIBUTES_FLAGS_NO_SCALE_WITH_STACK)
```

Comment:
we should initialize it with _NONE

Reply:
yeah that should of been `EFFECT_ATTRIBUTES_FLAGS_NONE`. Was definitely a mistake. I'll fix that